src/components/DownloadDataModal.js

```diff
@@ -44,8 +44,13 @@ const DownloadDataModal = ({ currentSelectedSite, sites }) => {
 
   const availableProtocols = useMemo(() => {
     const allSites = sites.map(site => site[1]).flat()
-    const allProtocols = allSites.map(site => Object.keys(site.protocols)[0])
-    const uniqueAvailableProtocol = [...new Set(allProtocols)]
+    const allProtocols = allSites.map(site => Object.keys(site.protocols)).flat()
+
+    // Both keys 'colonies_bleached' and 'quadrat_benthic_percent' are part of Bleaching protocol.
+    // this filter is based on the setup in transect-protocol.js as 'quadrat_benthic_percent' is used to determined as Bleaching.
+    const uniqueAvailableProtocol = [...new Set(allProtocols)].filter(
+      protocols => protocols !== 'colonies_bleached',
```

Comment:
would it work to use `protocols === 'quadrat_benthic_percent'` here to be more explicit?

Reply:
Here are list of protocols, both 'colonies_bleached' and 'quadrat_benthic_percent' do mean the same thing here, but I have to pick one to render this unique list. 

![Screenshot 2023-06-01 at 12 09 00 PM](https://github.com/data-mermaid/mermaid-dash/assets/6288799/17a1efa6-7738-4994-9369-5eba25142031)

Reply:
ok so for `uniqueAvailableProtocol` we are filtering out bleached records then? was just looking at the code comments

Reply:
it still filter all the protocols including bleaching, `colonies_bleached` is an **extra** key, but it also refers to Bleaching (I could pick either `colonies_bleached` or `quadrat_benthic_percent`)
it is confusing though :)

Reply:
ok thanks for explaining - approved in that case!